query/src/main/scala/filodb/query/PlanEnums.scala

```diff
@@ -75,6 +75,7 @@ object RangeFunctionId extends Enum[RangeFunctionId] {
   case object Timestamp extends RangeFunctionId("timestamp", Seq(InstantVectorParam()))
   case object AbsentOverTime extends RangeFunctionId("absent_over_time", Seq(RangeVectorParam()))
   case object PresentOverTime extends RangeFunctionId("present_over_time", Seq(RangeVectorParam()))
+  case object MedianAbsoluteDeviationOverTime extends RangeFunctionId("mad_over_time", Seq(RangeVectorParam()))
```

Comment:
Could you add a test into `PlannerHierarchySpec`? Nothing fancy -- just enough to show that the function is parsed correctly.